projects/plugins/jetpack/extensions/shared/memberships/edit.js

```diff
@@ -22,24 +20,6 @@ export const useAccessLevel = postType => {
 	return accessLevel;
 };
 
-export const accessOptions = {
```

Comment:
Anything that isn't static feels less of a "constant" to me so access options in this shared file felt better, but not a strong opinion.

Reply:
you mean is not static because of the translations?

Reply:
Yes, but I'm just being a semantic. :-)

Reply:
I like semantic too but I need it in a small file so the view bundle doesn't grow

Reply:
You reason is better! 👍 :-)